tox.ini

```diff
@@ -10,8 +10,27 @@ envlist =
 skip_missing_interpreters = {tty:True:False}
 
 
+[default]
+
+basepython = python3.8
+
+deps =
+    idna==2.9
+
+    test: typing==3.7.4.1
+    test: {[testenv:coverage_report]deps}
+    test-{py26,py27,py34}: pytest==4.6.9
+    test-{py35,py36,py37,py38}: pytest==5.2.4
+    test: pytest-cov==2.8.1
+
+setenv =
+    PY_MODULE=hyperlink
+
+    PYTHONPYCACHEPREFIX={envtmpdir}/pycache
+
+
```

Comment:
This default config is added here so that it can be referenced in multiple sections below.

Reply:
This consolidates versions, so we only need to fix them in one place, and helps `requires.io` which only knows how to fix them in one place.

Reply:
consolidation appreciated, by me and I'm sure the requires.io bot